hail_scripts/utils/clinvar.py

```diff
@@ -21,9 +40,68 @@
         "practice_guideline": 4,
     }
 )
+CLINVAR_PATHOGENICITIES_LOOKUP = hl.dict(
+    hl.enumerate(
+        # NB: sorted by pathogenicity
+        [
+            'Pathogenic',
+            'Pathogenic/Likely_pathogenic',
+            'Pathogenic/Likely_pathogenic/Likely_risk_allele',
+            'Pathogenic/Likely_risk_allele',
+            'Likely_pathogenic',
+            'Likely_pathogenic/Likely_risk_allele',
+            'Likely_risk_allele',
+            'Conflicting_interpretations_of_pathogenicity',
+            'Established_risk_allele',
+            'Uncertain_risk_allele',
+            'Uncertain_significance/Uncertain_risk_allele',
+            'Uncertain_significance',
+            'Likely_benign',
+            'Benign/Likely_benign',
+            'Benign',
+            CLINVAR_DEFAULT_PATHOGENICITY,
+        ],
+        index_first=False
```

Comment:
what does this do?

Reply:
flips the index of the enumerate to be second rather than first:

```
In [4]: hl.eval(hl.enumerate(['a', 'b', 'c']))
Out[4]: [(0, 'a'), (1, 'b'), (2, 'c')]

In [5]: hl.eval(hl.enumerate(['a', 'b', 'c'], index_first=False))
Out[5]: [('a', 0), ('b', 1), ('c', 2)]
```

makes the `hl.dict` call work!